examples/chip-tool/commands/clusters/ClusterCommand.h

```diff
@@ -107,6 +112,34 @@ class ClusterCommand : public ModelCommand, public chip::app::CommandSender::Cal
         return CHIP_NO_ERROR;
     }
 
+    template <class T>
+    CHIP_ERROR SendGroupCommand(chip::GroupId groupId, chip::FabricIndex fabricIndex, chip::NodeId senderNodeId,
+                                chip::ClusterId clusterId, chip::CommandId commandId, const T & value)
+    {
+        chip::app::CommandPathParams commandPath = { 0 /* endpoint */, groupId, clusterId, commandId,
+                                                     (chip::app::CommandPathFlags::kGroupIdValid) };
+
+        chip::Messaging::ExchangeManager * exchangeManager = chip::app::InteractionModelEngine::GetInstance()->GetExchangeManager();
+
+        auto commandSender =
+            chip::Platform::MakeUnique<chip::app::CommandSender>(this, exchangeManager, mTimedInteractionTimeoutMs.HasValue());
+        VerifyOrReturnError(commandSender != nullptr, CHIP_ERROR_NO_MEMORY);
+        ReturnErrorOnFailure(commandSender->AddRequestDataNoTimedCheck(commandPath, value, mTimedInteractionTimeoutMs));
+
+        chip::Optional<chip::SessionHandle> session =
+            exchangeManager->GetSessionManager()->CreateGroupSession(groupId, fabricIndex, senderNodeId);
+        if (!session.HasValue())
+        {
+            return CHIP_ERROR_NO_MEMORY;
+        }
+        CHIP_ERROR err = commandSender->SendGroupCommandRequest(session.Value());
+
+        commandSender.release();
```

Comment:
This is going to leak `commandSender` if `SendGroupCommandRequest` failed, no?

I think the logic here needs to be more like this:
```
    CHIP_ERROR err = commandSender->SendGroupCommandRequest(session.Value());
    exchangeManager->GetSessionManager()->RemoveGroupSession(session.Value()->AsGroupSession());
    ReturnErrorOnFailure(err);

    commandSender.release();

    return CHIP_NO_ERROR;
```